.travis.yml

```diff
@@ -29,3 +32,5 @@ after_success:
   # gover collects all .coverprofile files and saves it to one file gover.coverprofile
   - gover
   - goveralls -coverprofile=gover.coverprofile -service=travis-ci
+  # sync the docs only if everything else was successful
+  - cd script && ./sync-docs.sh
```

Comment:
@containscafeine @kadel 

Only reason why I do this is to make sure that the `cd kompose` and `cd ..` work correctly so there is absolutely no way to accident push to master by accident. Also to avoid copying `git clone` to a kompose directory that may be overridden/used in the future.